pymc/distributions/continuous.py

```diff
@@ -1399,6 +1405,12 @@ def dist(cls, lam, *args, **kwargs):
         # Aesara exponential op is parametrized in terms of mu (1/lam)
         return super().dist([at.inv(lam)], **kwargs)
 
+    def get_moment(rv, size, lam):
```

Comment:
We are converting lam to mu in the `dist` class method, so `get_moment` should expect mu already and you don't need to invert it (the same happens with the logcdf below). 

That's why your tests are failing for the exponential.

Reply:
Ah that makes sense, thanks! I'll make that change.